src/Microsoft.TestPlatform.ObjectModel/Logger/TestLoggerEvents.cs

```diff
@@ -39,6 +39,21 @@ protected TestLoggerEvents()
         /// </summary>
         public abstract event EventHandler<TestRunCompleteEventArgs> TestRunComplete;
 
+        /// <summary>
+        /// Raised when a discovery message is received.
+        /// </summary>
+        public abstract event EventHandler<TestRunMessageEventArgs> DiscoveryMessage;
```

Comment:
>TestRunMessageEventArgs [](start = 43, length = 23)

Why aren't we following the convention for this one?

Reply:
OnDiscoveryMessage sent by test platform sent TestRunMessageEventArgs object. Even though the naming is incorrect, TestLoggerEvents is just listening to what platform is exposing.